observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemDefinition.java

```diff
@@ -38,56 +42,61 @@
 
 class MicrometerSubsystemDefinition extends PersistentResourceDefinition {
     private static final String MICROMETER_MODULE = "org.wildfly.extension.micrometer";
-    private static final String MICROMETER_API_MODULE = "org.wildfly.extension.micrometer-api";
+    private static final String MICROMETER_API_MODULE = "org.wildfly.micrometer.deployment";
     static final String CLIENT_FACTORY_CAPABILITY = "org.wildfly.management.model-controller-client-factory";
-    static final String HTTP_EXTENSIBILITY_CAPABILITY = "org.wildfly.management.http.extensible";
     static final String MANAGEMENT_EXECUTOR = "org.wildfly.management.executor";
     static final String PROCESS_STATE_NOTIFIER = "org.wildfly.management.process-state-notifier";
+
     static final RuntimeCapability<Void> MICROMETER_COLLECTOR_RUNTIME_CAPABILITY =
             RuntimeCapability.Builder.of(MICROMETER_MODULE + ".wildfly-collector", MicrometerCollector.class)
                     .addRequirements(CLIENT_FACTORY_CAPABILITY, MANAGEMENT_EXECUTOR, PROCESS_STATE_NOTIFIER)
                     .build();
     static final RuntimeCapability<Void> MICROMETER_REGISTRY_RUNTIME_CAPABILITY =
             RuntimeCapability.Builder.of(MICROMETER_MODULE + ".registry", WildFlyRegistry.class)
                     .build();
-    static final RuntimeCapability<Void> MICROMETER_HTTP_CONTEXT_CAPABILITY =
-            RuntimeCapability.Builder.of(MICROMETER_MODULE + ".http-context", MicrometerContextService.class)
-                    .build();
-    static final RuntimeCapability METRICS_CAPABILITY =
-            RuntimeCapability.Builder.of("org.wildfly.management.http-context.metrics")
-                    .build();
     static final ServiceName MICROMETER_COLLECTOR = MICROMETER_COLLECTOR_RUNTIME_CAPABILITY.getCapabilityServiceName();
     static final String[] MODULES = {
-            "io.prometheus",
-            "org.latencyutils"
     };
+
     static final String[] EXPORTED_MODULES = {
             MICROMETER_API_MODULE,
+            "io.opentelemetry.otlp",
             "io.micrometer"
     };
 
-    static final AttributeDefinition SECURITY_ENABLED =
-            SimpleAttributeDefinitionBuilder.create("security-enabled", ModelType.BOOLEAN)
-                    .setDefaultValue(ModelNode.FALSE)
-                    .setRequired(false)
-                    .setRestartAllServices()
-                    .setAllowExpression(true)
-                    .build();
+    public static final SimpleAttributeDefinition ENDPOINT = SimpleAttributeDefinitionBuilder
+            .create(MicrometerConfigurationConstants.ENDPOINT, ModelType.STRING, true)
+            .setRequired(true)
```

Comment:
No need to set optional true in create(..) and then setRequired(true) :-) just use

`.create(MicrometerConfigurationConstants.ENDPOINT, ModelType.STRING)`

Reply:
Yeah, sorry. That was an oversight. I'll get that cleaned up.